VTEX - Logistics API.json

```diff
@@ -8850,9 +8452,195 @@
                     "type": "string",
                     "example": "application/json"
                 }
-            }
-        },
-        "schemas": {
+            }
+        },
+        "schemas": {
+            "ListPickupPointById": {
+                "type": "object",
+                "description": "Pickup point information.",
+                "properties": {
+                    "id": {
+                        "type": "string",
+                        "description": "[Pickup point](https://help.vtex.com/en/tutorial/pickup-points--2fljn6wLjn8M4lJHA6HP3R) ID."
+                    },
+                    "name": {
+                        "type": "string",
+                        "description": "Pickup point name displayed to customers at checkout."
+                    },
+                    "description": {
+                        "type": "string",
+                        "description": "Pickup point description displayed to customers at checkout."
+                    },
+                    "instructions": {
+                        "type": "string",
+                        "description": "Instructions to facilitate for customers when collecting their package."
```

Comment:
```suggestion
                        "description": "Instructions for customers when collecting their package."
```

Reply:
Where do these instructions appear for customers?

Reply:
I really think it is at checkout, but I'm assuming that. I will check that and change it later.